doc/contributing/releases.md

```diff
@@ -354,6 +352,12 @@ The new entry should take the following form:
 The release type should be either Current, LTS, or Maintenance, depending on the
 type of release being produced.
 
+By default, the `### Notable changes` section should be populated with the
+commits in the release that have either the `notable-change` or `semver-minor`
+label. Some `semver-minor` features may be determined by the releaser, or
+indicated by another contributor, to not be appropriate to be listed as a
+notable. The ultimate decision rests with the releaser.
```

Comment:
Should this paragraph be under "Notable changes" rather than "Commits"?

Looking at some of our changelog entries, sometimes we include "Other notable changes" as a subheading under "Notable changes" -- perhaps we could adopt this convention as the default place for commits labelled as `semver-minor` but not `notable-change`?

`semver-major` labelled commits would also be notable-by-default for semver-major releases, but not sure if that's better included here or in the separate section for major releases at the bottom of this doc.

Reply:
> Should this paragraph be under "Notable changes" rather than "Commits"?

Maybe it's the GH UI being confusing, but those headings are within the template. I opted to place this under the template as it seemed more readable as text rather than a bulleted list in the template. Not against moving it to within the template, though.

> Looking at some of our changelog entries, sometimes we include "Other notable changes" as a subheading under "Notable changes" -- perhaps we could adopt this convention as the default place for commits labelled as semver-minor but not notable-change?

Potentially. I'd prefer to keep this PR as a clarification in line with what matches the existing default template and output of the tooling, though. A conversation on changing the default recommended changelog structure should ideally be coupled with updates to both the template here and node-core-utils tooling - so I think it's best to agree/discuss changes to the defaults changelog structure in a nodejs/release thread where we can more easily set and track tasks to update both at the same time.

> semver-major labelled commits would also be notable-by-default for semver-major releases, but not sure if that's better included here or in the separate section for major releases at the bottom of this doc.

I think we have that covered in the major release docs already:

> For a major release, all SEMVER-MAJOR commits that are not strictly internal, test, or doc-related are to be listed as notable changes. Some SEMVER-MINOR commits may be listed as notable changes on a case-by-case basis. Use your judgment there.

Reply:
> Maybe it's the GH UI being confusing, but those headings are within the template. 

Yes, this. More obvious viewing the changed in preview mode as opposed to the "Files changed" tab. My apologies.